### ISSUE TITLE
cmake libprotobuf is missing libprotobuf.22.dylib

### ISSUE THREAD

**Issue by s22chan** [conda-forge/libprotobuf-feedstock#48]
Issue:
https://github.com/conda-forge/libprotobuf-feedstock/pull/47 breaks the `protobuf` package linkages:

```python
python -c 'import google.protobuf.json_format'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Users/steve/conda/envs/test/lib/python3.8/site-packages/google/protobuf/json_format.py", line 63, in <module>
    from google.protobuf import descriptor
  File "/Users/steve/conda/envs/test/lib/python3.8/site-packages/google/protobuf/descriptor.py", line 47, in <module>
    from google.protobuf.pyext import _message
ImportError: dlopen(/Users/steve/conda/envs/test/lib/python3.8/site-packages/google/protobuf/pyext/_message.cpython-38-darwin.so, 2): Library not loaded: @rpath/libprotobuf.22.dylib
  Referenced from: /Users/steve/conda/envs/test/lib/python3.8/site-packages/google/protobuf/pyext/_message.cpython-38-darwin.so
  Reason: image not found
```

Can we mark the package as bad or rebuild `protobuf` with a new dependency requirement. 

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list
# Name                    Version                   Build  Channel
ca-certificates           2019.11.28           hecc5488_0    conda-forge
certifi                   2019.11.28               py38_0    conda-forge
libcxx                    9.0.1                         1    conda-forge
libffi                    3.2.1             h6de7cb9_1006    conda-forge
libprotobuf               3.11.4               hd174df1_1    conda-forge
ncurses                   6.1               h0a44026_1002    conda-forge
openssl                   1.1.1d               h0b31af3_0    conda-forge
pip                       20.0.2                     py_2    conda-forge
protobuf                  3.11.4           py38h4a8c4bd_0    conda-forge
python                    3.8.1                hf2284b6_2    conda-forge
readline                  8.0                  hcfe32e1_0    conda-forge
setuptools                45.2.0                   py38_0    conda-forge
six                       1.14.0                   py38_0    conda-forge
sqlite                    3.30.1               h93121df_0    conda-forge
tk                        8.6.10               hbbe82c9_0    conda-forge
wheel                     0.34.2                     py_1    conda-forge
xz                        5.2.4             h1de35cc_1001    conda-forge
zlib                      1.2.11            h0b31af3_1006    conda-forge
```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

```
</details>

**Comment by jjhelmus** [comment]
The cmake build has a `libprotobuf.3.11.4.0.dylib` file rather than `lib/libprotobuf.22.dylib`.  I'm moving the build 1 packages to the broken label until this is resolved.

**Comment by wolfv** [comment]
argh! sorry about that! 

Would it be a good idea to consolidate the protobuf packages into one place and have one recipe with multiple outputs?

**Comment by s22chan** [comment]
Fixed #55
